homeassistant/components/matter/strings.json

```
@@ -563,6 +566,28 @@
           "description": "The Matter device to add to the other Matter network."
         }
       }
+    },
+    "water_heater_boost": {
+      "name": "Boost water heater",
+      "description": "Enables water heater boost for a specific duration.",
+      "fields": {
+        "duration": {
+          "name": "Duration",
+          "description": "Boost duration"
+        },
+        "emergency_boost": {
+          "name": "Emergency boost",
+          "description": "Whether to enable emergency boost mode."
+        },
+        "temporary_setpoint": {
+          "name": "Temporary setpoint",
+          "description": "Temporary setpoint temperature in Celsius during the boost period."
+        }
+      }
+    },
+    "water_heater_cancel_boost": {
+      "name": "Cancel water heater boost",
+      "description": "Cancel an ongoing water heater boost."
```

For consistency this should also use third-person singular, as we have "Enables"  for the other action we can use "Disables" here:

```suggestion
      "description": "Disables an ongoing water heater boost."
```